java/data/worldgen/attribute/mod.mcdoc

```diff
@@ -142,6 +142,11 @@ dispatch minecraft:environment_attribute[
 	"visual/sky_color",
 	"visual/sky_light_color",
 ] to RGBColorAttribute
+#[since="26.1"] dispatch minecraft:environment_attribute[
```

Comment:
The attribute shouldn't be inline and there should be a line gap between dispatchers